2022-07_sprint/index.md

```diff
@@ -1,3 +1,4 @@
+(current_sprint)=
 # July 2022 sprint
```

Comment:
@reshamas should we update all mentions of "sprint" to "open source working sessions"?

Reply:
How to rename the headers `Sprint info` and `Sprint events` in a short form?
- Something like: `OS session info`, or `Session info` and `OS events`  or simply `Events`

Then, `Sprint guide` and `Sprint preparation` would be `Session guide` and `Session preparation`?